tsconfig-strict.json

```diff
@@ -92,6 +92,7 @@
     "core/templates/components/forms/schema-based-editors/schema.constants.ts",
     "core/templates/components/on-screen-keyboard/on-screen-keyboard.component.ts",
     "core/templates/components/on-screen-keyboard/on-screen-keyboard.component.spec.ts",
+    "core/templates/components/question-directives/question-player/services/question-player-state.service.ts",
```

Comment:
Please add typing to the spec file too

Reply:
> Please add typing to the spec file too

Will add this. Thanks!

Reply:
Added the spec file `question-player-state.service.spec.ts`